docs/tutorials/timeslices.md

```diff
@@ -0,0 +1,258 @@
+<!--
+SPDX-FileCopyrightText: Copyright (c) 2024-2025 NVIDIA CORPORATION & AFFILIATES. All rights reserved.
+SPDX-License-Identifier: Apache-2.0
+-->
+
+# Time Slicing for Performance Analysis
+
+Time slicing allows you to analyze performance metrics across sequential time windows during a benchmark run. This feature provides visibility into performance trends, degradation patterns, and system behavior over time.
+
+## Overview
+
+Time slicing divides your benchmark into equal duration segments, computing metrics independently for each segment. This enables:
+
+- **Performance Trend Analysis**: Identify if performance degrades, improves, or stabilizes over time
+- **Warm-up Detection**: Distinguish initial cold-start behavior from steady-state performance
+- **Resource Exhaustion**: Spot gradual performance degradation due to memory leaks or resource pressure
+- **Load Pattern Impact**: Understand how different phases of load affect system performance
+- **Time-series Visualization**: Export data suitable for plotting performance trends
+
+## Core Parameters
+
+### Slice Duration
+- `--slice-duration MILLISECONDS`: Duration of each time slice
+- Recommended to be used with `--benchmark-duration`
+- Creates non-overlapping sequential time windows
+- Example: 10-second benchmark with 1000-millisecond slices creates 10 time windows
```

Comment:
technically I think grace-period could make this more than 10.

Reply:
Good point. Added a comment.